ui-tests/tools/com.liferay.ide.ui.fragment.tests/src/com/liferay/ide/ui/fragment/tests/NewFragmentFilesWizardMavenTests.java

```diff
@@ -16,24 +16,333 @@
+
+		wizardAction.openNewFragmentWizard();
+
+		wizardAction.prepareFragmentMaven(projectName);
+
+		wizardAction.next();
+
+		wizardAction.openBrowseOsgiBundleDialog();
+
+		dialogAction.prepareText("com.liferay.contacts.web");
+
+		dialogAction.confirm();
+
+		wizardAction.finish();
+
+		wizardAction.openLiferayModuleFragmentFilesWizard();
+
+		wizardAction.openAddOverrideFilesDialog();
+
+		dialogAction.selectItems("portlet.properties");
+
+		dialogAction.confirm();
+
+		wizardAction.finish();
+
+		String[] portletPropertiesFile = {projectName, "src", "main", "java", "portlet-ext.properties"};
+
+		Assert.assertTrue(viewAction.visibleProjectFileTry(portletPropertiesFile));
+
+		viewAction.closeAndDeleteProject(projectName);
 	}
 
+	@Test
+	public void addFragmentFilesResourceAction() {
+		String projectName = "test-fragment-files-resource-action-maven";
+
+		wizardAction.openNewFragmentWizard();
+
+		wizardAction.prepareFragmentMaven(projectName);
+
+		wizardAction.next();
+
+		wizardAction.openBrowseOsgiBundleDialog();
+
+		dialogAction.prepareText("com.liferay.dynamic.data.lists.web");
+
+		dialogAction.confirm();
+
+		wizardAction.finish();
+
+		wizardAction.openLiferayModuleFragmentFilesWizard();
+
+		wizardAction.openAddOverrideFilesDialog();
+
+		dialogAction.selectItems("resource-actions/default.xml");
+
+		dialogAction.confirm();
+
+		wizardAction.finish();
+
+		String[] resourceActionFile = {projectName, "src", "main", "resources", "resource-actions", "default-ext.xml"};
+
+		String[] portletPropertiesFile = {projectName, "src", "main", "resources", "portlet-ext.properties"};
+
+		Assert.assertTrue(viewAction.visibleProjectFileTry(resourceActionFile));
+
+		Assert.assertTrue(viewAction.visibleProjectFileTry(portletPropertiesFile));
+
+		viewAction.closeAndDeleteProject(projectName);
+	}
+
+	@Test
+	public void addFragmentFilesShortcuts() {
+		String projectName = "test-fragment-files-shortcuts-maven";
+
+		wizardAction.openNewFragmentWizard();
+
+		wizardAction.prepareFragmentMaven(projectName);
+
+		wizardAction.next();
+
+		wizardAction.openBrowseOsgiBundleDialog();
+
+		dialogAction.prepareText("com.liferay.journal.web");
+
+		dialogAction.confirm();
+
+		wizardAction.finish();
+
+		wizardAction.openLiferayModuleFragmentFilesWizard();
+
+		wizardAction.openAddOverrideFilesDialog();
+
+		dialogAction.selectItems("META-INF/resources/add_button.jsp");
+
+		dialogAction.confirm();
+
+		wizardAction.finish();
+
+		String[] jspFile = {projectName, "src", "main", "resources", "META-INF", "resources", "add_button.jsp"};
+
+		Assert.assertTrue(viewAction.visibleProjectFileTry(jspFile));
+
+		viewAction.openFileMenuFragmentFilesWizard();
+
+		wizardAction.openAddOverrideFilesDialog();
+
+		dialogAction.selectItems("META-INF/resources/article_vertical_card.jspf");
+
+		dialogAction.confirm();
+
+		wizardAction.finish();
+
+		String[] jspfFile = {projectName, "src", "main", "resources", "META-INF", "resources",
+		"article_vertical_card.jspf"
```

Comment:
Please Add the third test for Right click on a project > Liferay Module Fragment Files

Reply:
Also, this method should add into viewAction.java file, thanks

Reply:
Once we have this method, we'd like to change existing "wizardAction.openLiferayModuleFragmentFilesWizard()" to it.